docs/docs/configuration/index.md

```diff
@@ -380,6 +380,10 @@ timestamp_style:
 cameras:
   # Required: name of the camera
   back:
+    # Optional: Enable/Disable the camera (default: shown below).
+    #           If disabled: config is used but no live stream and no capture etc.
+    #           Events/Recordings are still viewable.
```

Comment:
```suggestion
    # Optional: Enable/Disable the camera (default: shown below).
    # If disabled: config is used but no live stream and no capture etc.
    # Events/Recordings are still viewable.
```

this is what I mean, the start of the comment is indented too much

Reply:
this still needs to be addressed

Reply:
a few lines above after thickness: 2 the optional block (and more of these above) is formatted in the same way. But I changed it anyway.